rules/DowngradePhp73/Rector/List_/DowngradeListReferenceAssignmentRector.php

```diff
@@ -309,7 +309,7 @@ private function createAssignRefWithArrayDimFetch(
     }
 
     /**
-     * @param array<ArrayItem|null> $arrayItems
+     * @param array<(ArrayItem | null)> $arrayItems
```

Comment:
This should be fixed in rector/rector probably

Reply:
noted 👍 , we can take a look later after migration ;), it seems happen during RenameClassRector, and touch docblock

Reply:
Good idea :) :+1: